libmscore/note.cpp

```diff
@@ -1011,6 +1011,18 @@ int Note::playTicks() const
       return (note->chord()->tick() + note->chord()->actualTicks() - stick).ticks();
       }
 
+//---------------------------------------------------------
+//   playTicksFraction
+///   Return total tick len of tied notes
+//---------------------------------------------------------
+
+Fraction Note::playTicksFraction() const
+      {
+      Fraction stick = firstTiedNote()->chord()->tick();
+      const Note* note = lastTiedNote();
+      return note->chord()->tick() + note->chord()->actualTicks() - stick;
+      }
```

Comment:
I think now you need to change the implementation of the `Note::playTicks` method so that there is no duplication
```
int Note::playTicks() const
{
    return playTicksFraction().ticks();
}
```

Reply:
Could you remove the duplication of code.